src/map/packets_struct.h

```diff
@@ -257,6 +257,13 @@ enum packet_headers {
 	rouletteinfoackType = 0xa1c,
 	roulettgenerateackType = 0xa20,
 	roulettercvitemackType = 0xa22,
+#if 0 // Unknown
```

Comment:
I think we can keep this as ifdeffed out until we support that client (it was added here just to remember about it later)